plugins/version.txt

```diff
@@ -1 +1 @@
-8
+10
```

Comment:
I have had to update two because I had two commits updating the `plugins.txt`

Reply:
Can you update it to back to 9 now